physics/Interstitials/UFS_SCM_NEPTUNE/sfcsub.F

```diff
@@ -16,14 +15,15 @@ module sfccyc_module
 !
       integer kpdtsf,kpdwet,kpdsno,kpdzor,kpdais,kpdtg3,kpdplr,kpdgla,
      &        kpdmxi,kpdscv,kpdsmc,kpdoro,kpdmsk,kpdstc,kpdacn,kpdveg,
-     &        kpdvet,kpdsot,kpdsoc,                                      
+     &        kpdlai, kpdvet,kpdsot,kpdsoc,                                      
      &        kpdvmn,kpdvmx,kpdslp,kpdabs
      &,       kpdsnd, kpdabs_0, kpdabs_1, kpdalb(4)
       parameter(kpdtsf=11,  kpdwet=86, kpdsno=65,  kpdzor=83,
 !    &          kpdalb=84,  kpdais=91, kpdtg3=11,  kpdplr=224,
      &          kpdais=91,  kpdtg3=11, kpdplr=224,
      &          kpdgla=238, kpdmxi=91, kpdscv=238, kpdsmc=144,
-     &          kpdoro=8,   kpdmsk=81, kpdstc=11,  kpdacn=91, kpdveg=87,
+     &          kpdoro=8,   kpdmsk=81, kpdstc=11,  kpdacn=91,
+     &          kpdveg=87,kpdlai=181,
```

Comment:
add a couple of spaces to line this up like above lines; also, why is this 181 and not 182? I see 182 as the grib number for LAI. is that what this is?

Reply:
Added spaces. Coudl you send me the link to where you saw the grib number please ?. I believe this could be a typo as i had copied this number from a hand written note from past discussions.

Reply:
https://www.nco.ncep.noaa.gov/pmb/docs/on388/table2.html